content/ja/docs/concepts/services-networking/ingress.md

```diff
@@ -81,51 +76,105 @@ Ingress [Spec](https://git.k8s.io/community/contributors/devel/sig-architecture/
 
 * オプションで設定可能なホスト名。上記のリソースの例では、ホスト名が指定されていないと、そのルールは指定されたIPアドレスを経由する全てのインバウンドHTTPトラフィックに適用されます。ホスト名が指定されていると(例: foo.bar.com)、そのルールはホストに対して適用されます。
 * パスのリスト(例: `/testpath`)。各パスには`serviceName`と`servicePort`で定義されるバックエンドが関連づけられます。ロードバランサーがトラフィックを関連づけられたServiceに転送するために、外部からくるリクエストのホスト名とパスが条件と一致させる必要があります。
-* [Serviceドキュメント](/ja/docs/concepts/services-networking/service/)に書かれているように、バックエンドはServiceとポート名の組み合わせとなります。Ingressで設定されたホスト名とパスのルールに一致するHTTP(とHTTPS)のリクエストは、リスト内のバックエンドに対して送信されます。
+* [Serviceドキュメント](/ja/docs/concepts/services-networking/service/)または{{< glossary_tooltip term_id="CustomResourceDefinition" text="CRD" >}}による[カスタムリソースバックエンド](#resource-backend)に書かれているように、バックエンドはServiceとポート名の組み合わせとなります。Ingressで設定されたホスト名とパスのルールに一致するHTTP(とHTTPS)のリクエストは、リスト内のバックエンドに対して送信されます。
```

Comment:
In Service resource, we specify target by service name and service port.
In custom resource backend, [by refering this file](https://github.com/kubernetes/website/pull/27328/files#diff-002dad180d6041d59804b40e53f5e6fdcd849efa7af758b825e59b1393740ad7), we specify neither service name nor service port.
IMHO, this translation needs to be update.

Reply:
updated this sentence for name and port combination are only applied for service.